extensions/default/src/validateInstances.ts

```diff
@@ -0,0 +1,51 @@
+import sortInstancesByPosition from '@ohif/core/src/utils/sortInstancesByPosition';
+import { constructableModalities } from '@ohif/core/src/utils/isDisplaySetReconstructable';
+import { DisplaySetMessage, DisplaySetMessageList } from '@ohif/core';
+import checkMultiFrame from './utils/validations/checkMultiframe';
+import checkSingleFrames from './utils/validations/checkSingleFrames';
+/**
+ * Checks if a series is reconstructable to a 3D volume.
+ *
+ * @param {Object[]} instances An array of `OHIFInstanceMetadata` objects.
+ */
+export default function validateInstances(
```

Comment:
It is great that you refactored the various validation methods and put them into the `validations` directory. However, some validation methods add messages to the messages list while others return `boolean`. Maybe we should name them accordingly so they are easy to identify which are which? This is just a suggestion. Not a big deal.

Reply:
Also, consider making the valdiation methods more typecript compliant.